packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -527,6 +538,20 @@ export class PerformanceEvent {
 		event.eventName = `${event.eventName}_${eventNameSuffix}`;
 		if (eventNameSuffix !== "start") {
 			event.duration = this.duration;
+			if (this.startMemoryCollection) {
+				const currentMemory = (performance as PerformanceWithMemory)?.memory
+					?.usedJSHeapSize;
+				const differenceInKBytes = Math.floor(
+					(currentMemory - this.startMemoryCollection) / 1024,
+				);
+				if (differenceInKBytes > 0) {
+					event.usedJSHeapSize = differenceInKBytes;
+				}
```

Comment:
Were we only interested to add this if it was a memory increase? In general I'd think there's a decent chance this number comes out negative and we won't record that; for Container load in particular it's probably not going to happen, but this being on an API we use in several places, seems like we'd want to know one way or the other?